client/app/house-info/page.tsx

```diff
@@ -3,18 +3,28 @@
 import React from 'react'
 import { Loading, PageContainer } from '@/_components/layout'
 import { Heading } from '@/_components/ui'
-import { useGetHouseQuery } from '@/_hooks/api'
-import { HouseInfoList } from './component'
+import { useGetMeQuery } from '@/_hooks/api'
+import { HouseInfoList, LeaveHouse } from './component'
 
 export default function HouseInfoPage() {
-  const { data: house, isLoading } = useGetHouseQuery()
+  const { data: user, isLoading } = useGetMeQuery()
 
-  if (isLoading || !house) return <Loading />
+  if (isLoading || !user) return <Loading />
+
+  const userHouse = user.houses
 
   return (
     <PageContainer>
-      <Heading title={`${house.name}'s House Info`} />
-      <HouseInfoList rules={house.rules} members={house.houseMembers} />
+      <Heading title={`${userHouse[0].name}'s House Info`} />
+      <HouseInfoList
+        rules={userHouse[0].rules}
+        members={userHouse[0].houseMembers}
+      />
+      {userHouse.length > 0 && (
```

Comment:
You don't need this condition because a user must have a house when the user accesses this page. Otherwise, they can not access this page. Controlled by `CheckAuth` file!

Reply:
Fixed it. 
1f3f9aff16a609c401618015940a5e482961df08